content/millennium/r4/base/entities/device.md

```diff
@@ -27,6 +31,8 @@ The following fields are returned if valued:
 * [Device name](https://hl7.org/fhir/r4/device-definitions.html#Device.deviceName){:target="_blank"}
   * [Name](https://hl7.org/fhir/r4/device-definitions.html#Device.deviceName.name){:target="_blank"}
   * [Type](https://hl7.org/fhir/r4/device-definitions.html#Device.deviceName.type){:target="_blank"}
+    * [User Friendly name](https://hl7.org/fhir/r4/codesystem-device-nametype.html#device-nametype-user-friendly-name){:target="_blank"}
+    * [Manufacturer name](https://hl7.org/fhir/r4/codesystem-device-nametype.html#device-nametype-manufacturer-name){:target="_blank"}
```

Comment:
These should be defined in the Terminology Table rather than here. See https://github.com/cerner/fhir.cerner.com/blob/main/lib/resources/r4/charge_item.yaml#L12 for an example from ChargeItem